backend/polls/views.py

```diff
@@ -92,6 +92,28 @@ def results(self, *args, **kwargs):
         data = {"poll": poll.id, "results": poll.results}
         return response.Response(data=data, status=status.HTTP_200_OK)
 
+    @action(detail=False, methods=("get",))
+    def stats(self, *args, **kwargs):
+        """Return:
+         - the number of polls with the REVIEWING (if the user is admin, otherwise None).
+         - the number of polls to which the user can vote."""
+        return response.Response(
+            data={
+                "number_of_pending_polls": len(Poll.objects.filter(state="REVIEWING"))
+                if self.request.user.is_staff
+                else None,
+                "number_of_available_polls": len(
+                    [
+                        poll
+                        for poll in Poll.objects.filter()
```

Comment:
We could get better perfs by actually using the `filter` method and the `count()`